{{ cookiecutter.package_name|slugify }}/.github/workflows/{% if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}deploy.yml{% endif %}

```diff
@@ -38,29 +38,27 @@ jobs:
       - name: Log in to the Docker registry
         uses: docker/login-action@v1
         with:
-          registry: {% raw %}${{ env.REGISTRY }}{% endraw %}
-          username: {% raw %}${{ github.actor }}{% endraw %}
-          password: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
```

Comment:
We need the `{% raw %}` bits here because otherwise Cookiecutter will treat these as values that should be interpolated.